OpenRA.Mods.Cnc/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -168,8 +168,7 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			metadata.AppendLine();
 
 			metadata.AppendLine("Terrain:");
-			terrainTypes = terrainTypes.Distinct().ToArray();
-			foreach (var terrainType in terrainTypes)
+			foreach (var terrainType in terrainTypes.ToHashSet())
```

Comment:
I'd use `terrainTypes.Distinct()` to avoid perturbing the output order here. But `ToArray()` seems safe to go.